src/pretix/control/templates/pretixcontrol/global_update.html

```diff
@@ -25,9 +25,9 @@
             <div class="alert alert-danger">
                 {% trans "The last update check was not successful." %}
                 {% if gs.settings.update_check_result.error == "http_error" %}
-                    {% trans "The pretix.eu server returned an error code." %}
+                    {% trans "The eventyay server returned an error code." %}
                 {% elif gs.settings.update_check_result.error == "unavailable" %}
-                    {% trans "The pretix.eu server could not be reached." %}
+                    {% trans "The eventyay server could not be reached." %}
```

Comment:
This is a good change, but should be in a different PR.
But ok, we can leave it hear as extra bonus ;-)